ui/src/pages/Upload.tsx

```diff
@@ -0,0 +1,212 @@
+import {
+  Button,
+  FormControl,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+} from "@material-ui/core/";
+import { DropzoneArea } from "material-ui-dropzone";
+import * as React from "react";
+import styled from "styled-components";
+
+import { device } from "../styles/device";
+// https://yuvaleros.github.io/material-ui-dropzone/
+
+const StyledGrid = styled(Grid)`
+  margin-top: 48px;
+  @media ${device.laptop} {
+    justify: left;
+    width: 100vw;
+  }
+`;
+
+export const Upload: React.FC = () => {
+  const [state, setState] = React.useState<{
+    storyTitle: string;
+    storySummary: string;
+    storyContent: string;
+    author: string;
+    authorBio: string;
+    images: File[];
+    country: string;
+    city: string;
+    year: string | number;
+    videoLink: string;
+  }>({
+    storyTitle: "",
+    storySummary: "",
+    storyContent: "",
+    author: "",
+    authorBio: "",
+    images: [],
+    country: "",
+    city: "",
+    year: "",
+    videoLink: "",
+  });
+
+  const setImage = (files: File[]) => {
+    setState({
+      ...state,
+      images: files,
+    });
+  };
+
+  const handleChange = (
+    event?: React.ChangeEvent<{ name?: string; value: unknown }>
+  ) => {
+    const name = event.target.name as keyof typeof state;
+    setState({
+      ...state,
+      [name]: event.target.value,
+    });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    const data = { state };
+    console.log(JSON.stringify(data));
+    fetch("localhost:8900/api/stories", {
+      method: "POST",
+      body: JSON.stringify(data),
+      headers: {
+        "Content-Type": "application/json",
+      },
+    })
+      .then((response) => response.json())
+      .then((response) => console.log("Success:", JSON.stringify(response)))
+      .catch((error) => console.error("Error:", error));
```

Comment:
let's open a TODO task to care of error state -> we should render something if we get a request error.